custom/icds_reports/utils/aggregation_helpers/distributed/agg_dashboard_activity.py

```diff
@@ -93,7 +95,7 @@ def get_user_locations(self):
 
             state_id, district_id, block_id, user_level = None, None, None, None
 
-            if user['location_id']:
+            if user['location_id'] and user['location_id'] in self.transformed_locations:
```

Comment:
don't include users who don't have a location assigned or assigned any other location which is not valid like test location etc.